openstef/feature_engineering/missing_values_transformer.py

```diff
@@ -57,6 +58,11 @@ def __init__(
             ).set_output(transform="pandas")
             self.imputer_._validate_params()
 
+    def remove_trailing_null_rows(self, df: pd.DataFrame) -> pd.DataFrame:
+        """Remove rows with trailing null values in a DataFrame."""
+        self.non_trailing_null_rows = ~df.bfill().isnull().any(axis="columns")
+        return df.loc[self.non_trailing_null_rows]
+
```

Comment:
I think that it would be nice to have a helper method to remove trailing null rows.
It would be nice if it was self contained (no refs to self). And private _remove_trailing_...

Also, I think it isn't used anywhere right?